source/MaterialXCore/Types.cpp

```diff
@@ -23,6 +23,7 @@ const string SURFACE_MATERIAL_NODE_STRING = "surfacematerial";
 const string VOLUME_MATERIAL_NODE_STRING = "volumematerial";
 const string MULTI_OUTPUT_TYPE_STRING = "multioutput";
 const string NONE_TYPE_STRING = "none";
+const string COMMENT_TYPE_STRING = "comment";
```

Comment:
Since this string represents a _node category_ rather than a data type, maybe the name of this new string constant should be `COMMENT_ELEMENT_CATEGORY`?

Alternatively, maybe we could just use the existing string constant `CommentElement::CATEGORY`?

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/source/MaterialXCore/Element.h#L1199

Reply:
I thought that CommentElement::CATEGORY was not accessible for some reason. I've removed the new string.